notanorm/base.py

```diff
@@ -290,6 +326,10 @@ def create_table(self, name, schema: DbTable):
 
     def execute(self, sql, parameters=()):
         with self.r_lock:
+            if self.__capture:
+                self.__capture_stmts.append((sql, parameters))
+                return FakeCursor()
```

Comment:
More of a feature request than a direct comment: it'd be cool to support simultaneously (really) executing and capturing. We have that internal test fixture that currently does that through monkeypatching, so proper support would be nice. That said, kinda tangential to the actual use-case here.

Reply:
can just add a `exec=True` param to the capture call